Tasks/DeployAzureResourceGroup/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -4,30 +4,30 @@
   "loc.description": "Deploy, start, stop, delete Azure Resource Groups",
   "loc.instanceNameFormat": "Azure Deployment:$(action) action on $(resourceGroupName)",
   "loc.group.displayName.output": "Output",
-  "loc.input.label.ConnectedServiceName": "Azure RM Subscription",
+  "loc.input.label.ConnectedServiceName": "Azure subscription",
```

Comment:
This string changes are from some previous commit which did not update resource files properly. Please ignore.